snipeit.sh

```diff
@@ -36,16 +35,16 @@ spin[1]="\\"
 spin[2]="|"
 spin[3]="/"
 
-rm -rf $tmp/
+rm -rf ${tmp:?}
```

Comment:
Not really, this prevents deleting ```/``` if ```$tmp``` is unset.

https://github.com/koalaman/shellcheck/wiki/SC2115

The colon is bash builtin fallback for unset variables.